assets/css/_styleWhitelabel.css

```diff
@@ -1 +0,0 @@
-/* Use this file to paste-in classes that you want to modify.*/
```

Comment:
This file is removed and did not get added to the UI repo. Do we not needing this file anymore?

Reply:
![image](https://user-images.githubusercontent.com/16466430/79148574-adeee300-7d93-11ea-903e-1e15c316f15d.png)

Reply:
We don't need it anymore. I just understood your question. It's an old version of themeWhiteLabel.

Reply:
Gotcha